tests/rkt_tests.go

```diff
@@ -96,6 +97,11 @@ func newRktRunCtx() *rktRunCtx {
 	}
 }
 
+func (ctx *rktRunCtx) launchMDS() error {
+	ctx.mds = exec.Command("../bin/rkt", "metadata-service")
+	return ctx.mds.Start()
```

Comment:
This function is not used at the moment.

But I think it is ok for now; it will be followed up by #1106.